ERCS/erc-interoperable-delegated-account.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 7779
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
Sure, updated the ERC number and the filename.